libbindgen/src/ir/enum_ty.rs

```diff
@@ -49,8 +49,11 @@ impl Enum {
         }
 
         let declaration = ty.declaration().canonical();
-        let repr = Item::from_ty(&declaration.enum_type(), None, None, ctx)
-            .ok();
+        let et = declaration.enum_type();
+        if et.is_none() {
+            return Err(ParseError::Continue);
+        }
+        let repr = Item::from_ty(&et.unwrap(), None, None, ctx).ok();
```

Comment:
Note that you need to run `cargo test` from the `libbindgen` directory, not the root one.